packages/cli/src/checkBeforeRunning.ts

```diff
@@ -0,0 +1,39 @@
+import {Hook} from '@oclif/config'
+import isBlitzRoot, {IsBlitzRootError} from './utils/is-blitz-root'
+import chalk from 'chalk'
+
+const whitelistGlobal = ['new']
+
+const hook: Hook<'init'> = async function (options) {
+  // Bug with oclif
+  const id = (options as any).Command.id
+  if (id && whitelistGlobal.includes(id)) return
+
+  const {err, message, depth} = await isBlitzRoot()
+
+  if (err) {
+    switch (message) {
+      case IsBlitzRootError.notBlitz:
+        this.log(
```

Comment:
can we use the `@blitzjs/server/utils/log` utility rather than the Command logger?